config.c

```diff
@@ -294,10 +313,11 @@ static enum parser_result parse_item(struct config *cfg,
 				     const char *option,
 				     const char *value)
 {
-	enum parser_result r;
 	struct config_item *cgi, *dst;
 	struct config_enum *cte;
+	enum parser_result r;
 	double df = 0.0;
+	uint64_t u64;
```

Comment:
This new u64 variable triggers -Wmaybe-uninitialized warning for me. Can you please initialize it?